code/game/objects/structures/popout_cake.dm

```diff
@@ -180,9 +180,17 @@
 /obj/structure/popout_cake/corpse_grabber/New()
 	..()
 
-	spawn(40)
+	if(ticker)
+		initialize()
```

Comment:
Couldn't this be called twice if the away mission gets loaded while there's a ticker?

Actually the ticket exists as soon as the subsystems begin initializations, so